src/libs/Growl.ts

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import CONST from '../CONST';
+
+type GrowlRef = {
```

Comment:
@BartoszGrajdek Just FYI that [we had some discussion here](https://github.com/Expensify/App/pull/26795#discussion_r1321560452) about something that is very similar to what we have here.

Reply:
Added a [comment](https://github.com/Expensify/App/issues/25127#issuecomment-1715359171) for future issue concerning this component 😄